Sources/TSCUtility/IndexStore.swift

```diff
@@ -14,9 +14,19 @@ import TSCBasic
 public final class IndexStore {
 
     public struct TestCaseClass {
+        public struct TestMethod: Hashable, Comparable {
+            public let name: String
+            public let isAsync: Bool
+
+            public static func < (lhs: IndexStore.TestCaseClass.TestMethod, rhs: IndexStore.TestCaseClass.TestMethod) -> Bool {
```

Comment:
Do you need full `Comparable` or just `Equatable`? `Equatable` should be fully synthesized.

Reply:
Lower down, we `sort` these, so I think we do actually need `Comparable`.